project-mode-line-tag.el

```diff
@@ -74,15 +74,21 @@ display.")
     (font-lock-append-text-property 0 (length project-tag) 'face 'project-mode-line-tag-project-tag project-tag)
     project-tag))
 
+(defvar-local project-mode-line-tag--not-in-project nil
```

Comment:
Suggest to call this `project-mode-line-tag--buffer-without-vc-project` with a docstring of “Non-nil if this buffer was found to not be part of a vc project. No further checks for vc project membership will be made once this variable is set.”

What do you think?